addons/sale/report/sale_report.py

```diff
@@ -58,13 +58,20 @@ def _get_done_states(self):
     volume = fields.Float('Volume', readonly=True)
 
     discount = fields.Float('Discount %', readonly=True)
-    discount_amount = fields.Float('Discount Amount', readonly=True)
+    discount_amount = fields.Monetary('Discount Amount', readonly=True)
+    currency_id = fields.Many2one('res.currency', compute='_compute_currency_id')
     campaign_id = fields.Many2one('utm.campaign', 'Campaign', readonly=True)
     medium_id = fields.Many2one('utm.medium', 'Medium', readonly=True)
     source_id = fields.Many2one('utm.source', 'Source', readonly=True)
 
     order_id = fields.Many2one('sale.order', 'Order #', readonly=True)
 
+    @api.depends_context('allowed_company_ids')
+    def _compute_currency_id(self):
+        user_company = self.env.company
+        user_currency = user_company.currency_id
+        self.currency_id = user_currency
```

Comment:
```suggestion
        self.currency_id = self.env.company.currency_id
```
Doesn't bring much added value to keep it in three lines IMHO